src/status_im2/contexts/communities/actions/request_to_join/view.cljs

```diff
@@ -28,19 +28,15 @@
                     requested-to-join-at]} (rf/sub [:get-screen-params])
             pending?                       (rf/sub [:communities/my-pending-request-to-join id])
             is-open?                       (not= 3 (:access permissions))]
```

Comment:
could be just `open?` instead

Reply:
Sure, though the PR's goal is to just fix bugs. I'll make this change ;)